### PR TITLE
new(config): added build-drivers jobs for bottlerocket.

### PR DESCRIPTION
**Pull request by FedeDP**
Signed-off-by: Federico Di Pierro <nierro92@gmail.com>

### CODE REVIEW

File: config/jobs/build-drivers/build-new-bottlerocket.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+presubmits:
+  falcosecurity/test-infra:
+  - name: validate-new-drivers-bottlerocket-4-presubmit
```
Comment (FedeDP) [inline config/jobs/build-drivers/build-new-bottlerocket.yaml:3]: We only have 4.X and 5.X kernels on bottlerocket.

**Comment by FedeDP** [comment]
Imho btw, i'd completely drop `rebuild-all-drivers` jobs, since they are useless in my opinion.
Moreover, i don't get why there is a presubmit in the `rebuild-all-drivers`, that triggers a publish :/

**Comment by FedeDP** [comment]
Note: we still need to bump driverkit to a version that is capable of building bottlerocket target ;)

**Comment by FedeDP** [comment]
> Note: we still need to bump driverkit to a version that is capable of building bottlerocket target ;)

Done ;)

**Review by maxgio92 — APPROVED** [review]
/approve

**Pull request merged**